packages/mobile/src/home/SendOrRequestBar.tsx

```diff
@@ -3,27 +3,27 @@ import Touchable from '@celo/react-components/components/Touchable'
 import QRCodeBorderlessIcon from '@celo/react-components/icons/QRCodeBorderless'
 import colors from '@celo/react-components/styles/colors.v2'
 import variables from '@celo/react-components/styles/variables'
-import React, { useCallback } from 'react'
+import React from 'react'
 import { useTranslation } from 'react-i18next'
 import { StyleSheet, View } from 'react-native'
 import { Namespaces } from 'src/i18n'
 import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
 
 export default function SendOrRequestBar() {
-  const { t } = useTranslation(Namespaces.sendFlow7)
-
-  const onPressSend = useCallback(() => {
+  function onPressSend() {
```

Comment:
We should be using arrow functions for inline functions right?

Reply:
if theres no "this" its seems less important. idk new debate?

Reply:
Well, I see I was the only one who voted for using `function` for this, so, it seems yes - will refactor it.